bin/asinit

```diff
@@ -6,8 +6,26 @@ const fs = require("fs");
 const path = require("path");
 const colors = require("../cli/util/colors");
 const version = require("../package.json").version;
+const options = require("../cli/util/options");
 
-if (process.argv.length != 3 || process.argv[2] == "--help" || process.argv[2] == "-h") printHelp();
+const asinitOptions = {
+  "help": {
+    "category": "General",
+    "description": "Prints a help message.",
+    "type": "b",
+    "alias": "h"
+  },
+  "yes": {
+    "category": "General",
+    "description": "Skips asking the user if they want to setup an AssemblyScript project.",
```

Comment:
I'd define this a little more future proof: "Answers all quetions with their default options for non-interactive usage."